app/build.gradle

```diff
@@ -43,6 +43,9 @@ dependencies {
     implementation 'com.dinuscxj:circleprogressbar:1.1.1'
     implementation 'com.karumi:dexter:5.0.0'
     implementation "com.jakewharton:butterknife:$BUTTERKNIFE_VERSION"
+
+    implementation 'com.squareup.picasso:picasso:2.5.2'
```

Comment:
We already have an image library, do we need Picasso for some specific purpose ?

Reply:
I didn't knew about that, which is the library the app currently uses?

Reply:
Fresco

Reply:
OK I'll remove this and use fresco

Reply:
We will have to remove the dependency then